torchrl/objectives/ppo.py

```diff
@@ -87,6 +88,8 @@ class PPOLoss(LossModule):
             Can be one of "l1", "l2" or "smooth_l1". Defaults to ``"smooth_l1"``.
         normalize_advantage (bool, optional): if ``True``, the advantage will be normalized
             before being used. Defaults to ``False``.
+        normalize_advantage_exclude_dims (Sequence[int], optional): dimensions to exclude from the advantage
+            standardization, can be negative. Useful in multiagent settings to exlude the agent dimension. Default: ().
```

Comment:
```suggestion
            standardization. Negative dimensions are valid. This is useful in multiagent settings where the agent dimension may be excluded from the reductions.
            Defaults to ``()``.
```

Reply:
I have a question reguarding the "negative" dims.

I guess ultimately we would also like to do the same with this
https://github.com/pytorch/rl/blob/d009835b4fccd1482e9a2bd2b597e00824b95c2d/torchrl/objectives/value/advantages.py#L696-L701

But there we have an extra "time" dimension, so we must be careful that `agent_dim=(-2,)` may actually mean `(-3,)` in some cases (similarly, `feature_dim=(4,)` may be `feature_dim=(5,)`).

Is it an issue in this (ie, PPO) case?

Reply:
I think the user should pass the dimension index for the tensor that goes in the class.

If you give as input [batch,time,agents,n] then agents is -2

also agents is usually after time